src/server/ApiRoutes/ApiRoutes.js

```diff
@@ -85,4 +90,5 @@ router
   .route('/')
   .get(MainApp);
 
+
```

Comment:
Another trivial thing, but best to avoid double blank lines (it's arbitrary I know)